src/modules/audio/audio.c

```diff
@@ -30,21 +36,18 @@ struct Source {
   bool playing;
   bool looping;
   bool spatial;
+  float pose[16];
```

Comment:
Usually the `pose` term is used for an 8-float position + quaternion, `transform` would be a more consistent name for a 16-float mat4 transform.